File: app/src/main/java/com/privat/contacts/ContactsApplication.java
package com.privat.contacts;

import com.privat.contacts.di.DaggerAppComponent;

import dagger.android.AndroidInjector;
import dagger.android.DaggerApplication;

public class ContactsApplication extends DaggerApplication {
    @Override
    protected AndroidInjector<? extends DaggerApplication> applicationInjector() {
        return DaggerAppComponent.factory().create(this);
    }
}

File: app/src/main/java/com/privat/contacts/ContactsApplication.kt
package com.privat.contacts

import com.privat.contacts.di.DaggerAppComponent
import dagger.android.AndroidInjector
import dagger.android.DaggerApplication

class ContactsApplication : dagger.android.DaggerApplication() {
    override fun applicationInjector(): AndroidInjector<out DaggerApplication> {
        return DaggerAppComponent.factory().create(this)
    }
}
